### DISCUSSION TITLE
removing locale slug

### DISCUSSION THREAD

**Discussion by ufukayyildiz** [requarks/wiki#2994] [Help / Questions] [answered]
hi

how can i remove **locale slug** from the url?

i want to use domain.com/blabla instead of domain.com/fr/blabla

Thank you in advance.

**Comment by NGPixel** [comment, marked as answer]
Make sure locale namespacing is disabled in the Admin > Locale page. You'll need to re-render all pages (under Utilities > Content) as well.

Note that there're still a few scenarios where the locale will be included in links. This is being addressed in the upcoming 3.0 release.

**Reply by KamilJurik** [reply]
Even having namespacing disabled the locale string is still included in the pages URL.
We still have the locale string included in the page URLs, ie: .../locale/page, instead of .../page  :(

**Reply by ufukayyildiz** [reply]
Thank you. An on this point i get _**Duplicate without user-selected canonical**_ on google search console. Because both **en/blabla** and **/blabla** is active and live. How this can be fixed. @NGPixel :)

**Comment by ufukayyildiz** [comment]
thanks.

When i remove tr ( turkish ) it changes to English automaticly and path turns to **en/** , i **can not remove it.**


_The locale code will be prefixed to all paths. (e.g. /en/page-name)
Paths without a locale code will be automatically redirected to the base locale defined above._


![image](https://user-images.githubusercontent.com/23309320/106939285-1e0a9300-6731-11eb-8678-870b089b671f.png)

![image](https://user-images.githubusercontent.com/23309320/106940769-003e2d80-6733-11eb-8828-39ff1999c0ce.png)


**not: my home page is without tr/ slug. But pages have tr/ slugs ( please see above imge ) and all pages also work without tr/ slug.**

**Reply by NGPixel** [reply]
This is a known bug that will be addressed in 3.0.

**Reply by KamilJurik** [reply]
Great. And the 3.0 time release is what?
Its blocking us using wiki...

**Reply by NGPixel** [reply]
https://docs.requarks.io/en/releases

**Reply by KamilJurik** [reply]
Great! Ty. You're doing awesome job!
